website/sdds-sbcom-docs/docs/_examples/_autocomplete/_cleaning-logic.mdx

```
@@ -0,0 +1,42 @@
+            }}
+          >
+            <IconClose color="inherit" size="s" />
+          </EmbedIconButton>
+        }
+       onChange={(e) => setValue(e.target.value)}
+       onSuggestionSelect={(e) => setValue(e.label)}
+      />
+    </div>
+  );
+}
+```
```

ESLint

[error] Insert `;`

(prettier/prettier)